### PR TITLE
ci: remove no-optional flag and disable cypress binary

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ ENV CYPRESS_INSTALL_BINARY=0
 WORKDIR /app
 COPY frontend/ .
 RUN if [ "$BUILDARCH" != "amd64" ]; then apt-get update && apt-get install -y build-essential python3 --no-install-recommends; fi
-RUN npm install -g pnpm@9 && pnpm install --no-optional --frozen-lockfile
+RUN npm install -g pnpm@9 && pnpm install --frozen-lockfile
```
Comment (yabirgb) [inline Dockerfile:10]: in docker don't you need to keep it? or set the environment variable too?

File: package.py [hunk 2]
```diff
@@ -96,6 +96,7 @@ def __init__(self) -> None:
         os.environ.pop(CERTIFICATE_KEY, None)
         os.environ.pop(APPLE_ID, None)
         os.environ.pop(APPLE_ID_PASS, None)
+        os.environ.setdefault('CYPRESS_INSTALL_BINARY', '0')
```
Comment (yabirgb) [inline package.py:99]: is this related to the removal of the no-optional?

Comment (kelsos) [reply package.py:99]: yes this stops the installation from packaging from downloading the Cypress binary when installing because we don't really need it, the whole idea behind --no-optional was to avoid downloading cypress on packaging and save some time.

File: Dockerfile (hunk 1 above)
Comment (kelsos) [reply Dockerfile:10]: It's already there in line 6 of the dockerfile so no need to set it

**Review by yabirgb — APPROVED** [review]
thanks kelsos. LGTM
